projects/muziehdesign/forms/src/lib/model-schema.factory.ts

```diff
@@ -160,4 +165,13 @@ export class ModelSchemaFactory {
 
     return nestedSchema;
   }
+
+  private buildFileSchema(options: FileTypeAnnotations) {
+    let schema = Yup.mixed().nullable().optional();
+    if (options.required) {
+      schema = schema.required(options.required.message);
```

Comment:
will you add a unit test to projects/muziehdesign/forms/src/lib/model-schema.factory.spec.ts please